.github/ISSUE_TEMPLATE/new-repo-checklist.md

```diff
@@ -0,0 +1,36 @@
+---
+name: New Repository Checklist
+about: Ensure all new MapLibre repositories confirm to our standards
+title: "Onboarding repository [repo]"
+
+---
+
+# Procedural
+- [ ] Any two board members must agree to accept a new repository. Names: [name1], [name2]
+
+# Licensing
+- [ ] The repo license is BSD-3 or MIT.  Repos may allow dual-licensing under other open source licenses like Apache.
+- [ ] The repo contains Copyright (c) <year> MapLibre contributors.
+- [ ] The repo has a `/LICENSE` file (could also have additional `LICENSE-MIT` / `LICENSE-APACHE` files as needed).
+
+# Other Files
+- [ ] The repo has `/CONTRIBUTING.md`
+- [ ] The repo has `/README.md`
+- [ ] The repo has `/CODE_OF_CONDUCT.md`
```

Comment:
```suggestion
- [ ] The repo has `/CODE_OF_CONDUCT.md`
- [ ] Pull Request and Issue Templates have been created
```

Reply:
I prepared some nicely looking ones here: https://github.com/maplibre/maplibre-rs/tree/main/.github

Maybe we can make them "official".